assets/js/app.js

```diff
@@ -0,0 +1,19 @@
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
```

Comment:
Please use and mind your naming conventions; For instance, why do you have to use `showSlides` can you please explain why you used it.

Reply:
Just a function to make the images slide
For this, I was trying it out and it didn't work for  as I expected forgot to delete it from my file
But I have removed it